mkosi/run.py

```diff
@@ -125,7 +126,8 @@ def become_root() -> tuple[int, int]:
     os.setresgid(0, 0, 0)
     os.setgroups([0])
 
-    return SUBRANGE - 100, SUBRANGE - 100
+    InvokingUser.uid = SUBRANGE - 100
+    InvokingUser.gid = SUBRANGE - 100
```

Comment:
Looking at this again could we maybe add a
```python
INVOKINGUSER = InvokingUser
```
to `util` and import the functionality via this? Otherwise there is no indication, that this is some global state that will be manipulated here, i.e. for a casual reader it's unclear that the once initialised `InvokingUser` will change at all.

Reply:
I will fix this in a follow up